pulsar/producer_partition.go

```diff
@@ -745,28 +933,34 @@ func (p *partitionProducer) failTimeoutMessages() {
 }
 
 func (p *partitionProducer) internalFlushCurrentBatches() {
-	batchesData, sequenceIDs, callbacks, errors := p.batchBuilder.FlushBatches()
+	batchesData, sequenceIDs, callbacks, errs := p.batchBuilder.FlushBatches()
```

Comment:
Why rename it?

Reply:
`Errors.Is()` is used to judge internal error. The original parameter name `errors` is conflict with package `errors`